app/templates/contact_us.html

```diff
@@ -4,22 +4,33 @@
 {%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
 {% from "partials/contact_details.html" import contact_details %}
 {% from "partials/migration_banner.html" import migration_banner %}
-{% set pageHeading %}{% trans %}Contact us{% endtrans %}{% endset %}
+{% set pageHeading %}{% trans %}Get help with your application{% endtrans %}{% endset %}
 {% block content %}
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
             {% if migration_banner_enabled %}
                 {{migration_banner()}}
             {% endif %}
-            <h1 class="govuk-heading-xl">{{pageHeading}}</h1>
+
             {% if fund_name %}
```

Comment:
@tferns check this https://github.com/communitiesuk/funding-service-design-fund-store/pull/295/files/32c70de32893ae3f9152f281757d5ff7416e1f6e#r1584597545

Reply:
I have replied there, thanks, let me know if it doesn’t make sense